drivers/SmartThings/matter-pump/profiles/pump-level.yml

```diff
@@ -0,0 +1,21 @@
+name: pump-level
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+  - id: pumpOperationMode
+    version: 1
+  - id: pumpControlMode
+    version: 1
+  - id: firmwareUpdate
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: Irrigation

```

Comment:
Shouldn't we use the Pump category @HunsupJung ?

Comment copied from here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1422#discussion_r1632061585

Reply:
HJ commented what the process was on following ticket.
https://smartthings.atlassian.net/browse/CHAD-13354?focusedCommentId=1456242

Reply:
I responded on the ticket and we can discuss further there

Reply:
I have updated the profiles to use the `Pump` category based on internal discussion here: https://smartthings.atlassian.net/browse/CHAD-13354?focusedCommentId=1456242